src/utils/aepp-wallet-communication/rpc/wallet-rpc.js

```diff
@@ -69,6 +69,33 @@ const REQUESTS = {
       }
     )
   },
+  [METHODS.bridge] (callInstance, instance, client, { name }) {
+    // Authorization check
+    if (!client.isConnected()) return { error: ERRORS.notAuthorize() }
+
+    // wait for approval
+    client.updateInfo()
+
+    return callInstance(
+      'onNodeBridge',
+      { name },
+      () => {
+        return {
+          result: {
+            url: instance.selectedNode.url,
+            ...(instance.selectedNode.internalUrl &&
+              {
+                internalUrl: instance.selectedNode.internalUrl
+              })
+          }
+        }
```

Comment:
```suggestion
        const { url, internalUrl } = instance.selectedNode
        return {
          result: { url, ...internalUrl && { internalUrl }) }
        }
```
I'm not sure about the last🤷‍♀️